api/modules/files/controller.js

```diff
@@ -73,12 +73,26 @@ class FilesController extends BaseController {
     .header(`Content-Type`, `application/octet-stream`);
   }
 
+  getOne(request, reply) {
+    const record = request.pre.records[0];
+
+    reply(
+      request.generateResponse(
+        this.formatResponseData(record)
+      ).code(200)
+    );
+  }
+
   create(request, reply) {
     return super.create(request, reply, formatResponse);
   }
 
   update(request, reply) {
-    return super.update(request, reply, formatResponse);
+    return Promise.fromCallback(next => {
+      return request.server.methods.file.cache.drop(request.params.id, next);
```

Comment:
While it's probably no big deal, we can safely drop the cache and not block the response to the client. Just a recommendation.